fe/fe-core/src/main/java/com/starrocks/common/util/PrintableMap.java

```diff
@@ -44,6 +44,8 @@ public class PrintableMap<K, V> {
         SENSITIVE_KEY.add("fs.s3a.secret.key");
         SENSITIVE_KEY.add("aws.s3.access_key");
         SENSITIVE_KEY.add("aws.s3.secret_key");
+        SENSITIVE_KEY.add("aws.glue.access_key");
```

Comment:
how about security info in Routine Load? And, what's the "client" here?
1. ssl.key.password
2. sasl.oauthbearer.client.secret

Should they be prefixed with "property."?